config/psp.example.json

```diff
@@ -0,0 +1,69 @@
+{
+  "chains": [
+    {
+      "id": "80001",
+      "contracts": {
+        "AirnodeProtocol": "0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8",
+        "DapiServer": "0xFF298a3169117606BDA78eD6C04917224968f9b5"
+      },
+      "allocators": [
+        {
+          "address": "0xFF298a3169117606BDA78eD6C04917224968f9b5",
+          "startIndex": "0",
+          "endIndex": "5"
+        }
+      ]
+    },
```

Comment:
Looks good to me (for when we'll use allocators). Alternatively we may want to have a list of indices for a bit more flexibility.